### PR TITLE
Fix some installation problems

### PR DESCRIPTION
**Pull request by runnersaw**
A few fixes for things that didn't work, as well as a change in the config file to remove specific installation details (the user's username). I think that we should prefer using relative paths to full paths that include the username -- the latter means that we would all have to modify that to make it work on our system.

### CODE REVIEW

**Review by jabb1123 — COMMENTED** [review]
These look great. Nothing to drastic

**Comment by segerphilip** [comment]
:shipit:

File: application/.application [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [info]
 default_project_dir = ../project_dir
-ti_install_dir = /home/jriedel/Traffic
+# If the next line gives problems, this could be changed to, for example, /home/svaughan/Traffic
+ti_install_dir = ~/Traffic
```
Comment (youralien) [inline application/.application:4]: Nice, works with Windows and UNIX!

**Review by youralien — APPROVED** [review]
Yay for making things useful in multiple contexts.  Next step would have Traffic Intelligence installed on a UNIX system, and try with these changes.

**Pull request merged**